extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "The SQL Database Projects extension for Azure Data Studio allows users to develop and publish database schemas.",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
should we update the engine to latest too. or just in gallery file will be enough?